CONTRIBUTING.md

```diff
@@ -58,3 +58,25 @@ import Foo as F
 import Qux (Q(..), q)
 import qualified A
 ```
+
+## Development Workflow
+
+Here we outline a general workflow new contributors could adopt for a nice incremental development experience.
+
+The root folder consists of two stack-*.yaml config files.
```

Comment:
There are more stack files. I think we could delete the lts12 one, the ffi-unescape is for replacing certain parts with a C implementation - perhaps it's not worth mentioning that one.

Reply:
I can just list the 3 we have and give some description of what they do. It could be helpful for anyone who would make changes that would require that they use those stack files.  

But, can I assume we will be removing the `lts12` file?

Reply:
Yeah, let's!